core/lib/refinery/plugin.rb

```diff
@@ -1,5 +1,16 @@
 module Refinery
   class Plugin
+    META = {
```

Comment:
No, they don't need to be here.  I was just my preference to see and edit them all in one place, especially when I have their position here as well (it will be in the next PR).  This list then becomes a sort of the skeleton of where all other menu items fit in.

For example, I think we should define something like this for menu positions 1-40:

- 1-10 = site content (pages, blog, stories, faqs, ...)
- 11-19 = site specific items or slightly lower priority content
- 20 = default
- 21-30 = forms (inquiries, job applications, mailing lists, ...)
- 31-40 = people (users, members, leadership team, "my profile", ...)